src/edge/templates/edge/edge.html

```diff
@@ -60,6 +60,9 @@
         g.event = function(i, v, s) {
           g("event", { n: i, p: v }, s);
         };
+        g.anonymize = function() {
```

Comment:
I suppose this comes with the fs template?

Reply:
Yup.